src/assg07-library.cpp

```diff
@@ -75,6 +93,12 @@ void swap(int& a, int& b)
   b = temp;
 }
 
+void swap(char& a, char& b)
```

Comment:
You were asked/required to create the function documentation for the task 2 and 3 functions, was not done here.